src/borg/lrucache.py

```diff
@@ -28,6 +28,14 @@ def __delitem__(self, key):
     def __contains__(self, key):
         return key in self._cache
 
+    def get(self, key, default=None):
+        value = self._cache.get(key, default)
+        if value is default:
+            return value
```

Comment:
That doesn't really make sense (I do know the sentinel pattern), since the LRUCache is specifically meant to cache *something* not *Nonething*.